softwarereview_editor_management.pt.Rmd

```diff
@@ -0,0 +1,102 @@
+We ask that editors make an informal commitment of serving for two years, and re-evaluate their participation after that.  
+On a short-term basis, any editor can decline to handle a package or say, "I'm pretty busy, I can't handle a new package for a few weeks."
+
+In addition to handling packages, editors weigh in on group editorial decisions, such as whether a package is in-scope, and determining updates to our policies. 
+We generally do this through Slack, which we expect editors to be able to check regularly. 
+We have editorial board calls annually.  
+We also rotate Editor-in-Chief responsibilities (first-pass scope decisions and assigning editors) amongst the board about quarterly. 
+You'll have the opportunity to enter this rotation once you have been on the board for some time, usually at least six months. 
+Some of us also take on bigger projects to improve the peer-review process, though this is entirely optional. 
+
+We hope that you'll join the board!  
+It's an exciting time for peer-review at rOpenSci.
+
+Please give this some thought, ask us any questions you have, and let us know whether you can join us.
+
+Best,
+[EDITOR], on behalf of the rOpenSci Editorial Board
+```
+
+## Integração de um novo editor {#onboarding-a-new-editor}
+
+- Informe o gerente da comunidade rOpenSci para que
+  
+  - editores sejam adicionados à lista de [site da rOpenSci](https://github.com/ropensci/roweb3/#team-member).
+  - Você pode criar uma postagem de introdução no blog.
+
+- Se eles ainda não o fizeram como editores convidados, solicite que o novo editor ative o [a autenticação de dois fatores (2FA) para o GitHub](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa).
+
+- Convide editores para a organização do rOpenSci no GitHub como membro, como membro da equipe de editores do rOpenSci. [`editors` equipe](https://github.com/orgs/ropensci/teams/editors) e da equipe [`data-pkg-editors`](https://github.com/orgs/ropensci/teams/data-pkg-editors) ou [`stats-board`](https://github.com/orgs/ropensci/teams/stats-board) ou subequipe, conforme apropriado.  Isso dará a eles as permissões apropriadas e permitirá que recebam notificações específicas da equipe.
+
+- Os editores precisam acessar o banco de dados AirTable de revisão de software.
+
+- Os editores precisam ter acesso ao canal privado de editores no espaço de trabalho do Slack da rOpenSci (e ao espaço de trabalho do Slack em geral, caso não o tenham feito anteriormente; nesse caso, peça ao gerente da comunidade da rOpenSci).
+
+- Publique uma mensagem de boas-vindas no canal, enviando um ping para todos os editores.
+
+- No espaço de trabalho do Slack, eles precisam ser adicionados à equipe de editores para que `@editors` você também os contatará.
```

Comment:
Essa frase parece que precisa ser reformulada. Eu escreveria assim 'No espaço de trabalho do Slack, eles precisam ser adicionados a equipe de editores para que `@editors` os possam contatar também.'

Reply:
Another option is present in Pedro's review. It reads as a more straightforward explanation.